README.md

```diff
@@ -250,13 +250,18 @@ type NavigationBarIcon = {
         urlDark?: string;
     };
     badge?: {
-        /** Boolean to determine if the badge should be shown */
+        /**
+         * Boolean to determine if the badge should be shown
+         * If `show` is `true` and number and nativeLogic are not present, the badge will be shown as a dot
+         */
         show: boolean;
         /** Same logic and current same supported values as in nativeLogic field from API */
         nativeLogic?: 'INBOX' | 'PROFILE';
         /** Hardcoded value to set as the badge count. It will have more priority than nativeLogic. */
         number?: number;
     };
+    /** Tracking properties to be sent to analytics when the icon is clicked */
+    trackingProperties?: Record<string, string>;
```

Comment:
why not trackingEvent?

Reply:
This is not a TrackingEvent. Just properties that will be merged to the Tracking Event produced by the native side